modules/programs/starship.nix

```
@@ -24,14 +24,7 @@ in {
     };
 
     settings = mkOption {
-      type = with types;
-        let
-          prim = either bool (either int str);
-          primOrPrimAttrs = either prim (attrsOf prim);
-          entry = either prim (listOf primOrPrimAttrs);
-          entryOrAttrsOf = t: either entry (attrsOf t);
-          entries = entryOrAttrsOf (entryOrAttrsOf entry);
-        in attrsOf entries // { description = "Starship configuration"; };
+      type = with types; attrsOf anything;
```

https://nixos.org/manual/nixos/unstable/#sec-settings-options

```suggestion
      type = tomlFormat.type;
```

---

Good idea. I changed to this.